python/hail/tests/test_methods.py

```diff
@@ -429,13 +429,56 @@ def test_split_multi_hts(self):
             hl.split_multi_hts(ds)
 
     def test_mendel_errors(self):
-        dataset = self.get_dataset()
-        men, fam, ind, var = hl.mendel_errors(dataset, hl.Pedigree.read(resource('sample.fam')))
-        men.select('fam_id', 's', 'code')
-        fam.select('pat_id', 'children')
-        self.assertEqual(list(ind.key), ['s'])
-        self.assertEqual(list(var.key), ['locus', 'alleles'])
-        dataset.annotate_rows(mendel=var[dataset.locus, dataset.alleles]).count_rows()
+        mt = hl.import_vcf(resource('mendel.vcf'))
+        ped = hl.Pedigree.read(resource('mendel.fam'))
+
+        men, fam, ind, var = hl.mendel_errors(mt['GT'], ped)
+
+        self.assertEqual(fam.count(), 2)
+        self.assertEqual(ind.count(), 7)
+        self.assertEqual(var.count(), mt.count_rows())
+
+        self.assertEqual(set(fam.select('pat_id', 'mat_id', 'errors', 'snp_errors').collect()),
+                         {
+                             hl.utils.Struct(pat_id='Dad1', mat_id='Mom1', errors=41, snp_errors=39),
+                             hl.utils.Struct(pat_id='Dad2', mat_id='Mom2', errors=0, snp_errors=0)
+                         })
```

Comment:
Do you need an explicit check for the contents of `men`?

Reply:
ah, I think I meant to do that and never did. The scala tests weren't testing it!